main.go

```diff
@@ -208,7 +208,7 @@ func runCLIApp(c *cli.Context) (err error) {
 	}
 
 	if flags.Foreground {
-		err = logger.InitLogFile(flags.LogFile, flags.LogFormat)
+		err = logger.InitLogFile(flags.LogFile, flags.LogFormat, mountConfig.Severity)
```

Comment:
Have we checked the before and after behaviour of debug_fs, debug_http, debug_invariants and debug_mutex?
As I can only see the check for DebugFuse and DebugGCS in line 206.

Reply:
1. `debug_fuse_errors` flag writes error logs. The default logging level is INFO, which means error logs will be written in this case until and unless the user explicitly sets logging severity to off in the config file.
2. `debug_fs` and `debug_http` do not write any logs.
3. `debug_invariants` does not enable any logs.
4. `debug_mutex` flag just writes one log (code copied below). I don't think we should explicitly enable logs if this flag is set, but let me know what you think. 
```
d.logger.Printf("Potential dead lock detected for a lock %q held by: %v\n", d.name, d.holder)
```